src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from "prop-types";
+import { CarouselProvider, Slider, Slide, DotGroup } from "pure-react-carousel";
+import React from "react";
+
+import "pure-react-carousel/dist/react-carousel.es.css";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quotes }) => {
+  const Carousel = () => (
+    <CarouselProvider totalSlides={quotes.length} isIntrinsicHeight>
+      <Slider>
+        {quotes.map((dataObj, index) => (
+          <Slide index={index} key={dataObj.quote}>
+            <figure className={s.innerWrapper}>
+              <blockquote className={s.quote}>
+                &ldquo;{dataObj.quote}&rdquo;
+              </blockquote>
+              <figcaption className={s.attribution}>
+                &mdash;{dataObj.attribution}
+              </figcaption>
+            </figure>
+          </Slide>
+        ))}
+      </Slider>
+      <DotGroup className={s.dots} />
+    </CarouselProvider>
+  );
+
+  return (
+    <div className={s.bleedWrapper}>
+      <div className={s.bleedBackground} />
+      <div className={s.gridParent}>
+        <div className={s.gridAreaMiddle}>
```

Comment:
On Figma, isn't it aligned from column 1 to column 12 or is that the outer grid's maximum width?

Reply:
It is aligned from column 1–12, but because it spans the whole width and doesn't make use of the internal columns, you don't actually need the column definitions.